aot/src/main/java/com/dylibso/chicory/compiler/internal/WasmAnalyzer.java

```diff
@@ -82,18 +80,19 @@ public List<AotInstruction> analyze(int funcId) {
             AnnotatedInstruction ins = body.instructions().get(idx);
 
             if (labels.contains(idx)) {
-                result.add(new AotInstruction(AotOpCode.LABEL, idx));
+                result.add(new Instruction(OpCode.LABEL, idx));
             }
 
             // skip instructions after unconditional control transfer
             if (exitBlockDepth >= 0) {
                 if (ins.depth() > exitBlockDepth
-                        || (ins.opcode() != OpCode.ELSE && ins.opcode() != OpCode.END)) {
+                        || (ins.opcode() != com.dylibso.chicory.wasm.types.OpCode.ELSE
```

Comment:
Here you are using `com.dylibso.chicory.wasm.types` does this means that we can remove `aot/src/main/java/com/dylibso/chicory/compiler/internal/OpCode.java` ?

Reply:
I don't think so.  But it does highlight that maybe I should call the OpCode in the internal package CompilerOpCode to avoid the clash with the types one.

Reply:
renamed.

Reply:
:+1: